server_src/player.js

```diff
@@ -834,9 +834,10 @@ class Player {
         if (tick % 2 != 0 || squaredDist(p, this) > square(512)) return;
 
         // cooldown to prevent chat spam when 2 people are on the planet
-        const cool = p.cooldown;
-        if (cool < 0) {
+        let cool = p.cooldown;
```

Comment:
These don't look like style changes but rather a bugfix so I guess I'll let it slide